### PR TITLE
{lang}[foss/2019a] R v3.6.0

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@fizwit To make the tests pass here, we should get PR #8297 merged first for `NLopt` (which is waiting for some requested changes)

File: easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,2123 @@
+name = 'R'
+version = '3.6.0'
+biocver = '3.9'
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb]: @fizwit `biocver` isn't used anywhere, can be removed?

Comment (fizwit) [reply easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb]: I’ll remove it, but for locally we use it for adding BioConductor packages. Would be nice to support biocver in the future.

Comment (fizwit) [reply easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb]: removed,

File: easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb [hunk 2]
```diff
@@ -1962,7 +1961,7 @@ exts_list = [
         'checksums': ['6b82edf9fc7d1f98d5632bc94476bd10b39134a7bb267120b5e979a4590e195f'],
     }),
     ('hdf5r', '1.2.0', {
-        'preinstallopts': "unset LIBS && ",
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb]: @fizwit Please keep the `preinstallopts`, you should have both...

Comment (fizwit) [reply easybuild/easyconfigs/r/R/R-3.6.0-foss-2019a.eb]: preinstallopts has been re-installed

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node3129.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/24486f2ca90f24e9b43368175a31e01e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2692.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/73549c69299f33a89ecb48f28d4f61b5 for a full test report.

**Comment by boegel** [comment]
@fizwit I think the failing test reports are due to the missing preinstallopts for hdf5r (I started the test before you added your last commit).
I’ll restart the tests ASAP

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
node3171.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/43c48968e985b71ea7b64f929096c4ee for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2003.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4060ec43dbb90fc59ef8f1a884bf2195 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2416.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/793e2223b069ece8d11dd8e25421fad3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @fizwit!

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 8 out of 8 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/43a56760359b44174df518b3530a3218 for a full test report.

**Pull request merged**